test/test_nn.py

```diff
@@ -15587,7 +15587,7 @@ def test_embedding_backward(self, device, dtype):
         self.assertEqual(embedding.weight.grad._indices(), tensorTwice)
         self.assertEqual(embedding.weight.grad._values(), onesTwice)
 
-    @dtypesIfCUDA(*ALL_TENSORTYPES2)
+    @dtypesIfCUDA(*get_all_fp_dtypes(include_bfloat16=AMPERE_OR_ROCM))
```

Comment:
This would just be `include_bfloat16=TEST_WITH_ROCM`, right? The `TEST_WITH_ROCM` variable may need to be imported.